src/frontends/pytorch/src/op/add.cpp

```diff
@@ -12,12 +15,14 @@ namespace pytorch {
 namespace op {
 
 OutputVector translate_add(NodeContext& context) {
+    auto lhs = context.get_input(0);
```

Comment:
please check a number of inputs to the operation

Reply:
@eaidova adds a function for that in parallel PR, I will later update all operations to check number of inputs, so lets leave this out of scope for now